custom/opm/beneficiary.py

```diff
@@ -759,33 +765,44 @@ class ConditionsMet(OPMCaseRow):
         ('cash', _("Payment Amount"), True),
         ('case_id', _('Case ID'), True),
         ('closed_date', _("Closed On"), True),
-        ('payment_last_month', _("Payment amount received last month"), True),
+        ('payment_last_month', _("Payment amount received last month (Rs.)"), True),
         ('cash_received_last_month', _("Cash received last month"), True),
     ]
 
     def __init__(self, case, report, child_index=1, **kwargs):
         super(ConditionsMet, self).__init__(case, report, child_index=child_index, **kwargs)
         self.serial_number = child_index
-        self.payment_last_month = self.last_month_row.total_cash if self.last_month_row else 0
+        self.payment_last_month = "Rs.%d" % self.last_month_row.total_cash if self.last_month_row else 0
         self.cash_received_last_month = self.last_month_row.vhnd_available_display if self.last_month_row else 'no'
+        self.awc_code = CommCareUser.get_by_user_id(case.user_id).user_data.get('awc_code', EMPTY_FIELD) if \
+            case.user_id else EMPTY_FIELD
```

Comment:
I agree with @sravfeyn on this point: 
> This will require lot of individual couch calls. It would be ideal to get this from the result of Fluff Calculations. You can potentially use [this](https://github.com/dimagi/commcare-hq/pull/5834/files#diff-a7e5dd5cadeed32505efbf20c57da3e4R94) or any other [existing](https://github.com/dimagi/commcare-hq/blob/master/custom/opm/filters.py#L14) [Datawrappers](https://github.com/dimagi/commcare-hq/blob/master/custom/opm/reports.py#L1131) based off the fluff

All cases from a given AWC should have the same user, so this would be a lot of duplicate couch calls as well.  Each case row gets instantiated with the report class, so if you save some sort of data structure saved on the parent report class (probably on `CaseReportMixin`), you'll have access to that in every row without needing to perform duplicate queries.